From 4237cc165cd7447d6798559506adfd339a638dc0
From: Will Newton <will.newton@imgtec.com>
Date: Fri, 29 Jun 2012 13:48:09 +0000
Subject: libc/inet/resolv.c: Use IE model for TLS data.

All other TLS data in uClibc is explicitly using the IE model,
so use it for __resp too. This generates smaller and faster
code.

Signed-off-by: Will Newton <will.newton@imgtec.com>
Signed-off-by: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
---
diff --git a/libc/inet/resolv.c b/libc/inet/resolv.c
--- a/libc/inet/resolv.c
+++ b/libc/inet/resolv.c
@@ -3592,7 +3592,7 @@ struct __res_state _res __attribute__((section (".bss"))) attribute_hidden;
  *        relocations.
  */
 extern __thread struct __res_state *__libc_resp
-       __attribute__ ((alias ("__resp"))) attribute_hidden;
+       __attribute__ ((alias ("__resp"))) attribute_hidden attribute_tls_model_ie;
 # else
 #  undef __resp
 struct __res_state *__resp = &_res;
--
cgit v0.9.1
